src/PointwiseFunctions/AnalyticSolutions/GrMhd/AlfvenWave.hpp

```diff
@@ -117,23 +126,25 @@ class AlfvenWave {
     static type lower_bound() noexcept { return 1.0; }
   };
 
-  /// The strength of the background magnetic field.
-  struct BackgroundMagField {
-    using type = double;
+  /// The background static magnetic field vector.
+  struct BkgdMagneticField {
```

Comment:
I think you can name the class `BackgroundMagneticField` and specify a custom option name. Then you can make the variable name `backgroun_...` as well.